extensions/markdown-basics/syntaxes/markdown.tmLanguage.json

```diff
@@ -3001,7 +3001,7 @@
 					"name": "punctuation.definition.raw.markdown"
 				}
 			},
-			"match": "(`+)((?:[^`]|(?!(?<!`)\\1(?!`))`)*+)(\\1)",
+			"match": "(`+)((?:[^`]|(?!(?<!`)\\1(?!`))`)*+)(\\1)?",
```

Comment:
Changes to the grammar need to be made upsteam: https://github.com/Microsoft/vscode-markdown-tm-grammar

Also not sure we want this. A single backtick on a line should not be highlight the rest as inline code